README.md

```diff
@@ -1 +1,171 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] Great work implementing your professional README file 💯. However, please kindly ensure to remove the Microverse logo from here🤓.